nbr.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "nbr",
+  "release": "11.2-RELEASE",
+  "artifact": "https://github.com/NAKIVO/iocage-plugin-nbr.git",
+  "properties": {
+    "allow_socket_af": "1",
+    "allow_set_hostname": "1",
+    "allow_raw_sockets": "1",
+    "allow_mount_devfs": "1",
+    "allow_mount_procfs": "1",
+    "mount_devfs": "1",
+    "mount_fdescfs": "1",
+    "mount_procfs": "1",
+    "boot": "on"
```

Comment:
Let’s not make plugins start at boot unless users ask for that. Huge POLA violation when that’s not the way any other plugin or jail functions.

Reply:
Agreed on the boot on.